### ISSUE TITLE
Refine calculation of pairwise ephys methodology distances

### ISSUE THREAD

**Issue by stripathy** [neuroelectro/neuroelectro_neurotree#9]
I need to make sure I understand how curated info from article methods sections are converted into pairwise methodology differences. My sense is that in the current implementation, categorical variables get weighted more than continuous variables. 

Another idea is to weight the methodology variables by their relative predictive power for ephys differences. So electrode type would get a larger weight than external solution differences. @rgerkin do you have a sense for this?

**Comment by svdavid** [comment]
Didn't we discuss the possibility of not computing a single pairwise distance for methodology?  Not saying that the discussion isn't relevant, but we can also ask, for example, how much does NT distance explain differences in each methods variable.  

Alternative; You could come up with a "natural" weighting, depending on how much each correlates with NT distance and/or ephy differences.

stephen

**Comment by rgerkin** [comment]
I would prefer not to compute pairwise methodology differences until we have to.  I don't think any way of doing it will be very robust for doing the predictions we want to do.  When we do (eventually), the relative predictive power approach seems reasonable.

**Comment by stripathy** [comment]
@rgerkin but isn't a goal of our project to figure out "how are research methodologies passed down from adviser to trainee to trainee's trainee"? 

I agree that there's a ton of terrible ways to quantify pairwise methodology differences (in particular, lumping all the differences into a single number), but I think trying to keep them separate should be fine.

**Comment by rgerkin** [comment]
@stripathy I agree that is a goal, but don't think that is a necessary step right now, since establishing predictive power in a mostly model-free way is more important to demonstrating what connections are worth further exploring.
